src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutComponents.cs

```diff
@@ -48,39 +84,28 @@ public int CompareTo(IsEnabled? other)
         }
 
         private readonly IDisposable _activationDisposable;
-        private readonly IDisposable? _idsObservable;
 
-        public IsEnabled(
-            ValueComponent<bool?> valueComponent,
-            LoadoutItemId itemId,
-            bool isLocked)
+        public EnabledStateToggle(
+            ValueComponent<bool?> valueComponent)
         {
             _valueComponent = valueComponent;
-            _ids = new[] { itemId };
-            _isLockedComponent = new ValueComponent<bool>(value: isLocked);
 
             _activationDisposable = this.WhenActivated(static (self, disposables) =>
             {
                 self._valueComponent.Activate().AddTo(disposables);
             });
         }
 
-        public IsEnabled(
+        public EnabledStateToggle(
             ValueComponent<bool?> valueComponent,
-            IObservable<IChangeSet<LoadoutItemId, EntityId>> childrenItemIdsObservable,
-            ValueComponent<bool> isLockedComponent)
+            IObservable<IChangeSet<LoadoutItemId, EntityId>> childrenItemIdsObservable)
```

Comment:
This is unused and can be removed. The entire constructor can be removed.

Reply:
It seems to be used though

Reply:
It's passed as state into `WhenActivated` below, but the state is never used.

Reply:
Ah, I see what you mean